src/PhpParser/Printer/BetterStandardPrinter.php

```diff
@@ -505,6 +529,33 @@ private function detectTabOrSpaceIndentCharacter(array $stmts): void
         }
     }
 
+    private function cleanUpDuplicateContent(string $content): string
```

Comment:
This might looks like a random piece of code. Reference to the problem would help understand.


Could you refer the issue here?

Reply:
I've added  `@see https://github.com/rectorphp/rector/issues/4499` comment to reference issue.